src/ORM/DataList.php

```diff
@@ -1548,11 +1545,9 @@ public function sum($fieldName)
 
 
     /**
-     * Returns the first item in this DataList (instanceof DataObject)
+     * Returns the first item in this DataList
      *
      * The object returned is not cached, unlike {@link DataObject::get_one()}
-     *
-     * @return DataObject|null
```

Comment:
`@return T|null`?

Same with `last()` and `find()`?

Reply:
https://github.com/silverstripe/silverstripe-framework/pull/11108#discussion_r1451866912